tools/msg-bump-version.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-useless-escape */
+
+const msg = `
+
+      ,-.           ,.---'''^\                  O
+      {   \       ,__\,---''''''-.,      O    O
+      |    \     ','^           _  '..     o
+      \  ,.J..-''          // (O)   ,,X,    o
+      /  (_               ((   ~  ,;:'''  o
+      /   ,.X'.,            \\      ':;;;
+      (_../      -._                 ,,'
+                K.=,;.__ /^~/___..'''
+                        /  /'
+
+  Don't forget to bump the version in the package files!
```

Comment:
Let's update and link the docs. I could use help w/ the phrasing
```suggestion
  Don't forget to bump the version in the package files!
  Every PR should increase the package file versions 
  by either a patch, minor, or major number,
  based on the label on the issue being addressed.
  See contributing docs:  
  https://github.com/cuttle-cards/cuttle/blob/main/docs/CONTRIBUTING.md#make-changes
 
```